docs/markdown/glossary.md

```diff
@@ -0,0 +1,260 @@
+
+## Exchange Rate
+
+The price of one ledger's asset in terms of another ledger's asset. Routers may generate revenue from the difference in value between incoming and outgoing transfers. The exchange rate between a sender and a receiver is determined by the product of exchange rates at each hop.                                                                         
+
+## Fulfillment 
+
+Also known as the "preimage" or "hash preimage", a fulfillment is  a 32-byte value used to trigger the execution of a transfer. In most Interledger payments, the fulfillment is known only to the receiver (or in the case of the STREAM protocol it is known to the sender and the receiver).                                                                        
+
+## Hop 
+
+The movement of an ILP packet from one router to another as it travels through the Interledger network to reach its destination.                                                                         
+
+## IL-RFC  
+
+Interledger request for comments. A document describing a part of the protocol stack, or a topic related to it, such as this glossary. The official list of IL-RFCs is maintained in gh:interledger/rfcs.                                                                                                                               
+
+## Interledger (in "Let's use Interledger for that!")  
+
+The Interledger protocol stack.                                                                         
+
+## interledger (in "An interledger network")
+
+A network of two or more ledgers that use the Interledger protocol stack. 
+
+See also: [Interledger (The)](#interledger-in-the-interledger) 
+
+The adjective 'interledger' is also sometimes used to refer to transactions that cross multiple ledgers, even if ILP is not used (similar to 'international' meaning 'across multiple nations').                                                                         
+
+## Interledger (in "The Interledger")  
+
+The public network of ledgers connected via the Interledger protocol stack.                                                                         
+
+## Interledger Address 
+
+An Interledger address provides a way to route ILP packets to their intended destination through a series of hops, including any number of ILP routers. 
+
+An Interledger address is a dot-separated string that contains prefixes to group ledgers. An example address might look like `g.us.acmebank.acmecorp.sales.199` or `g.crypto.bitcoin.1BvBMSEYstWetqTFn5Au4m4GFg7xJaNVN2`. 
+
+
+Note that anyone can claim to have a certain Interledger address, there is no registry of them. Whether or not a payment ends up at the intended receiver is ultimately safeguarded by the hashlock condition, not by enforcement of address ownership. 
```

Comment:
there's ongoing discussion of a secure routing protocol which would actually allow address ownership. Not sure where active discussion on this is happening so I don't know where we can link it to

Reply:
I brought up secure routing during the ILP call yesterday - consensus on the call was to form a working group and work on secure routing as a "slow burn" since it's not urgent while the network is very nascent. Until we have some sort of PKI in Interledger, I think for now we can move forward using the current model of ILP address allocation.